src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -35,7 +35,8 @@
 #pragma region Widgets
 
 // clang-format off
-enum {
+enum
+{
```

Comment:
This file contains a ton of formatting changes that are unrelated. Please revert these.